### DISCUSSION TITLE
Remove version number from the Github artifacts

### DISCUSSION THREAD

**Discussion by vtrenton** [istio/istio#49672] [General]
Hey! I'm wondering if there is a reason the version number is included in the releases for istio and istioctl?

The reason im asking is because I like to pull the latest packages using githubs 'latest' url.

for example: https://github.com/istio/istio/releases/latest/istioctl-$ver-linux-amd64.tar.gz is a quick way to get the latest version of istioctl. it would be much easier to download whatever the current latest is if I didn't have to know the version in the actual artifact name.

Thanks!

**Comment by howardjohn** [comment]
It seems plausible if we change this we would break other users. And publishing both would be fairly expensive  and confusing.


`latest` may not always be safe, I think if we patch an old version it may become the "latest". Like if we publish 1.19.8 today, it would be 'latest' but it should be 1.20.3.

We have a script that handles this: https://istio.io/latest/docs/setup/getting-started/#download

**Reply by vtrenton** [reply]
Hey Howard,
Thanks for the reply on this! I agree that it could break existing workflows for users and that wouldn't be good. I like to download istioctl as the standalone bin instead of the entire istio release tarball. But it's not thaaaat much of an inconvience haha. I'll close this out then.
Thanks again for your time! :)

**Reply by howardjohn** [reply]
We also publish istioctl as a docker image, you can pluck it out with tools like https://github.com/google/go-containerregistry/blob/main/cmd/crane/doc/crane.md if its easier

**Reply by vtrenton** [reply]
That's a great idea! thanks for that suggestion! :)
